client/cluster_error.go

```diff
@@ -21,7 +21,11 @@ type ClusterError struct {
 }
 
 func (ce *ClusterError) Error() string {
-	return ErrClusterUnavailable.Error()
+	s := ce.Detail()
```

Comment:
there might be an extra \n at the end of the error. check line https://github.com/coreos/etcd/pull/4503/files#diff-de2e13ec6e63b9429e41a409355ad5d4R34